plugins/modules/vmware_dvswitch.py

```diff
@@ -23,7 +23,7 @@
 - Abhijeet Kasurde (@Akasurde)
 - Christian Kotte (@ckotte)
 notes:
-    - Tested on vSphere 6.5 and 6.7
+    - Tested on vSphere 6.5, 6.7 and 7.0
```

Comment:
Could you please add a changelog entry [here](https://github.com/ansible-collections/community.vmware/tree/main/changelogs/fragments)?